src/Workspaces/Core/Portable/CodeStyle/NotificationOption.cs

```diff
@@ -9,6 +9,7 @@ namespace Microsoft.CodeAnalysis.CodeStyle
     /// <remarks>
     /// This also supports various properties for databinding.
     /// </remarks>
+    /// <completionlist cref="NotificationOption"/>
```

Comment:
It makes it so that when you have "BlahBlah.Notification == $$" it preselects NotificationOption. This is useful for types that have a bunch of static members of that type.